Classes/DataProcessing/LanguageMenuProcessor.php

```diff
@@ -317,42 +300,29 @@ public function validateConfiguration()
             }
         }
         if (!empty($invalidArguments)) {
-            throw new \InvalidArgumentException('LanguageMenuProcessor Configuration contains invalid Arguments: ' . implode(', ', $invalidArguments), 1478806566);
+            throw new \InvalidArgumentException('LanguageMenuProcessor configuration contains invalid arguments: ' . implode(', ', $invalidArguments), 1522959188);
```

Comment:
error code should be regenerated to not to be equal with the core

Reply:
should be streamlined